Assets/Editor/UnitTests/Models/SchedulerEditorTest.cs

```diff
@@ -38,20 +38,18 @@ public void Init()
         if (FunctionsManager.ScheduledEvent == null)
         {
             new FunctionsManager();
-            FunctionsManager.ScheduledEvent.LoadScript(LuaFunctionString, "ScheduledEvent");
         }
 
+        FunctionsManager.ScheduledEvent.LoadScript(LuaFunctionString, "ScheduledEvent");
+
         if (PrototypeManager.SchedulerEvent == null)
         {
             new PrototypeManager();
```

Comment:
Unity doesn't like your suggestion. But we could just instantiate it without checking, since the constructor only creates instances of PrototypeMap, and on the next line it loads it.